src/kvilib/ext/KviCryptEngine.cpp

```diff
@@ -109,13 +109,13 @@
 		Obviously (with the current implementations) all the conversation endpoints
 		must agree on the engine (or better algorithm) used and on the key(s).
 		The key is user specified, so you have to find a secure way to negotiate
-		it with your communication engpoints. If you can meet the persons in the "real life",
-		this is the best way to exchange the keys, otherwise you can use mail & pgp.
-		Yes, this is a "miss" of the crypt protocol: it is missing a public key handshake.[br]
+		it with your communication endpoints. If you can meet the persons in the "real life",
+		this is the best way to exchange the keys, otherwise you can use mail & PGP.
+		Yes, this is a shortcoming of the crypt protocol: it is missing a public key handshake.[br]
 
 		[big]The first test[/big][br]
-		A cool way to test a crypting engine is to use a "self query": connect to
-		any irc server, and execute [cmd]query[/cmd] <yournickname>; a query window
+		A cool way to test a encrypting engine is to use a "self query": connect to
```

Comment:
`an` not `a`. Also unless `encrypting engine` is the proper name of it, I'd use `encryption engine` since it flows better.